src/runtime/shell.c

```diff
@@ -0,0 +1,291 @@
+#include "shell.h"
+
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
+
+#include "create_block.h"
+#include "constants.h"
+#include "wallet.h"
+#include "blockchain.h"
+#include "mempool.h"
+
+#define DELIMS " \t\n"
```

Comment:
I just imported as I needed them. I think its split between header and implementation properly, but I can take a second look and clean it up.

(I believe the standard we are setting based on conversation with Steve is: Import in header if definitions needed for header, import in implementation otherwise.)